project/docker-compose.yml

```diff
@@ -21,8 +21,7 @@ services:
       - backend
     environment:
       QUESTION_SERVICE_HOST: ${QUESTION_SERVICE_HOST}
-      SUPABASE_URL: ${API_GATEWAY_SUPABASE_URL}
-      SUPABASE_KEY: ${API_GATEWAY_SUPABASE_KEY}
+      USER_SERVICE_HOST: ${USER_SERVICE_HOST}
```

Comment:
> NOTE: NODE_ENV has to be set to production in docker, so that we can correctly resolve this if condition, because on docker the host should be the process.env.HOST variable instead of localhost. Open to alternatives for these cause this feels quite wrong

I think that's fine, so we can add set the NODE_ENV variable to be `production` here right?